### PR TITLE
[292] Correct ticket README

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -673,38 +673,42 @@ As stated, this type of object discovery requires some extra study and effort, b
...
 >>> from irods.ticket import Ticket
 >>> new_ticket = Ticket (granting_session)
->>> ticket_string = new_ticket.issue('read', '/zone/home/my/collection_with_data_objects_for/somebody').string
+>>> The_Ticket_String = new_ticket.issue('read', 
+...     '/zone/home/my/collection_with_data_objects_for/somebody').string
 
-At this point, another user (including the anonymous user) could then apply the
-ticket (knowing only its identifying string) to any session object and thus
-access the intended object(s):
+at which point that ticket's unique string may be given other users, who can then apply the
```
Comment (trel) [inline README.rst:694]: "given to"

File: README.rst [hunk 2]
```diff
@@ -673,38 +673,42 @@ As stated, this type of object discovery requires some extra study and effort, b
...
+ticket to any existing session object in order to gain access the intended object(s):
```
Comment (trel) [inline README.rst:695]: "access to"

**Comment by d-w-moore** [comment]
Ready for merge

**Comment by trel** [comment]
squash to one commit - add the #

**Comment by d-w-moore** [comment]
done!

**Pull request merged**